### DISCUSSION TITLE
large force error points

### DISCUSSION THREAD

**Discussion by llmxq** [deepmodeling/deepmd-kit#2448] [Q&A]
When i was training a deep model, i noticed points with large force error. as shown by the figure below. i suppose this is caused by some unreasonable data points, that the force of every atom isn't cauclated very well.  I think these points should be excluded in the training. is there any way to locate these pionts?
![1680969560098](https://user-images.githubusercontent.com/130238105/230731019-9937b159-87e7-4fdc-b4fc-9f62c2eecd9f.jpg)

**Comment by njzjz** [comment]
Please see #553
